assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -235,9 +237,23 @@ function APIProvider({ children }) {
     [link]
   );
 
-  const getAllUsers = useCallback(() => {
-    return apiFetch({ path: addQueryArgs(users, { per_page: '-1' }) });
-  }, [users]);
+  const getAuthors = useCallback(
+    (search = null) => {
+      return apiFetch({
+        path: addQueryArgs(users, { per_page: '100', who: 'authors', search }),
+      });
+    },
+    [users]
+  );
+
+  const getUserById = useCallback(
```

Comment:
What is function called? I can't see it used anywhere.

Reply:
yeah I don't think it's needed anymore and can be removed

Reply:
On it.